src/dataflow/src/source/kinesis.rs

```diff
@@ -195,9 +191,8 @@ impl SourceInfo<Vec<u8>> for KinesisSourceInfo {
                             // todo@jldlaughlin: Parse this to determine fatal/retriable?
                             error!("{}", e);
                             self.shard_queue.push_back((shard_id, shard_iterator));
-                            activator.activate();
```

Comment:
Why is it that we can remove these `activate()` calls. The caller of `get_next_message()` seems to also not be calling it.

This is really only for my understanding. 😅

Reply:
so now we return `TransientDelay` after which point the caller will call `activate` ( from within the `handle_source` function. its a bit of a 3 step process but i think this is a clearer api for `get_next_message`